indextts/infer.py

```
@@ -1,670 +1,497 @@
+        
+        use_deepspeed = False
+        if self.device_config.use_fp16:
             try:
                 import deepspeed
-
                 use_deepspeed = True
-            except (ImportError, OSError, CalledProcessError) as e:
-                use_deepspeed = False
-                print(f">> DeepSpeed加载失败，回退到标准推理: {e}")
-                print("See more details https://www.deepspeed.ai/tutorials/advanced-install/")
+            except ImportError: pass
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)